src/Controls/src/Core/Platform/iOS/Extensions/FormattedStringExtensions.cs

```diff
@@ -124,5 +126,116 @@ public static class FormattedStringExtensions
 
 			return attrString;
 		}
+
+		public static void RecalculateSpanPositions(this UILabel control, Label element)
```

Comment:
This comes from Xamarin.Forms, although in .NET MAUI I see an incorrect calculation of the Region that defines the limits of each Span when there are multiple lines.